_include/autoload.php

```diff
@@ -0,0 +1,20 @@
+<?php
+  spl_autoload_register(function ($class_name) {
+    if(preg_match('/^Keyman\\\\Site\\\\com\\\\keymanweb\\\\(.+)/', $class_name, $matches)) {
```

Comment:
We don't have any files there, hence the .keep file. But the infrastructure is in place for additional files later (and then we can delete the .keep file).